streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -936,6 +1047,60 @@ private InternalTopicConfig createInternalTopicConfig(final StateStoreSupplier s
     }
 
     /**
+     * Get the Pattern to match all topics requiring to start reading from earliest available offset
+     * @return the Pattern for matching all topics reading from earliest offset, never null
+     */
+    public synchronized Pattern earliestResetTopicsPattern() {
```

Comment:
Not sure if I can follow. "move them all to TopologyBuilder" ? It is in TopologyBuilder already...